packages/inter-protocol/package.json

```diff
@@ -63,7 +63,6 @@
     "import-meta-resolve": "^2.2.1"
   },
   "files": [
-    "bundles",
```

Comment:
@dckc and I have talked about publishing bundles but haven't though through it fully.

(@michaelfig mentioned that including it here adds 20MB, making it too big for repos that have a limit, like verdaccio at 10MB)